src/hotspot/os/linux/cgroupV1Subsystem_linux.cpp

```diff
@@ -81,7 +79,6 @@ jlong CgroupV1MemoryController::uses_mem_hierarchy() {
 }
 
 void CgroupV1MemoryController::set_subsystem_path(char *cgroup_path) {
```

Comment:
preexisting: This should be const char* (I was wondering if this modifies the memory handed in; it doesn't, but a non-const address really looks like an output parameter)